README.md

```diff
@@ -95,37 +100,65 @@ using a git repository and
     | Error message | Remedy |
     | --- | --- |
     | ERROR: StatsCollector: rpc error: code = PermissionDenied desc = The caller does not have permission | Ensure the authenticating user has at least the "Monitoring Editor" role |
+    | invalid_grant: reauth related error (invalid_rapt)     | gcloud authentication expired, run  `gcloud auth application-default login` |
+
+1.  Configure metrics
+
+    *   Datadog: Configure metrics by getting your
+        [API and application keys](https://docs.datadoghq.com/api/?lang=go#overview)
+        from Datadog and copying over your Datadog queries into `metrics.yaml`
+    *   New Relic: Configure data by providing
+
+        *   [API key](https://docs.newrelic.com/docs/apis/get-started/intro-apis/types-new-relic-api-keys)
+        *   [endpoint domain and path](https://docs.newrelic.com/docs/apis/rest-api-v2/getting-started/introduction-new-relic-rest-api-v2#appid)
+        *   [metric name and metric value](https://rpm.newrelic.com/api/explore/applications/metric_data?application_id=296573398&account=2334394)
+
+    *   Your `metrics.yaml` file should now look something like the following. See [below](#metricsyaml-configuration) for configuration details.
+
+```
+datadog_metrics:
+  - name: your_first_metric_name
+    query: "your metric query (copied from your Datadog dashboard)"
+    api_key: aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
+    application_key: bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb
+    destination: stackdriver
+  - name: your_second_metric_name
+    query: "your metric query (copied from your Datadog dashboard)"
+    api_key: aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
+    application_key: bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb
+    destination: stackdriver
+newrelic_metrics:
+  - name: total_call_time
+    endpoint_base: "https://api.newrelic.com"
+    endpoint_path: "/v2/applications/12345678/metrics/data.json"
+    metric_data: {
```

Comment:
This is pretty unconventional (even if valid) YAML. I'd suggest to remove the `{}` brackets and trailing spaces.